index.pug

```diff
@@ -0,0 +1,591 @@
+                li Collect in-store card payments
+                li Get detailed payment insights
+                li Backed by the safety &amp; security of Razorpay
+              span Explore &gt;
+          .sec7box1
+            a(href='https://razorpay.typeform.com/to/FuLkWeaT#email=xxxxx' target='_blank')
+              svg.svg-corner.svg-desktop(xmlns='http://www.w3.org/2000/svg' width='316' height='350' fill='#0f1b38')
+                path.border(d='M.5 219.511V6A5.5 5.5 0 016 .5h216.883c4.105 0 8.042 1.628 10.948 4.527l77.116 76.937a15.499 15.499 0 014.553 10.973v190.054a5.501 5.501 0 01-5.509 5.5l-304-.48a5.5 5.5 0 01-5.491-5.5z' stroke='#4aeccb')
+              img.lazy.boxicon(data-src="/assets/education/taponglass.svg", alt="taponglass")
+              h2 Tap on Glass
+              ul
+                li Collect card payments using your NFC android smartphone
+                li Let customers tap their credit &amp; debit cards to pay
+              span Sign up for early access &gt;
+// section 8 start
+section#section8.section8
+  h1 Why our clients choose us?
+  .sec8inside
+    .boxone1
+      .box1left1
+        img.lazy(data-src="/assets/education/Instantactivation.svg", alt="instant-activation")
+      .box2right1
+        span Instant Activation
+        |  Complete only onboarding. Get activated &amp; start transacting in minutes
+    .boxone1
+      .box1left1
+        img.lazy(data-src="/assets/education/Internationalpayments.svg", alt="international-payments")
+      .box2right1
+        span International payments
+        |  Take your business global &amp; reach international buyers with no additional integration costs
+    .boxone1
+      .box1left1
+        img.lazy(data-src="/assets/education/100paymentmodes.svg", alt="international-payments")
+      .box2right1
+        span 100+ payment modes
+        |  The most extensive set of payment methods with UPI, Wallets, Credit &amp; Debit cards, Netbanking, EMIs and PayLater
+    .boxone1
+      .box1left1
+        img.lazy(data-src="/assets/education/Robustsecurity.svg", alt="Robustsecurity")
+      .box2right1
+        span Robust security
+        |  PCI DSS Level 1 compliant along with frequent third party audits. 
+        a(href='https://razorpay.com/links/secure-payments-using-razorpay' target='_blank') Learn more
+    .boxone1
+      .box1left1
+        img.lazy(data-src="/assets/education/Dashboardreporting.svg", alt="dashboardreporting")
+      .box2right1
+        span Dashboard reporting
+        |  Real-time data and insights on payments, settlements, refunds to make informed decisions
+    .boxone1
+      .box1left1
+        img.lazy(data-src="/assets/education/Transparentpricing.svg", alt="transparent-pricing")
+      .box2right1
+        span Transparent pricing
+        |  No set up or maintenance fee. Pay only when you get paid!
+script(async, defer, src='js/custom.js' type='text/javascript')
```

Comment:
Please add end section, signup card code, please follow the guidelines which i have given you |

  section.endsection
    +endsection-plain-svg('#001E96','#001E96')
    .container
      h2.puck-ter.colored Get exciting features with Razorpay MultiQR
      p.size-md.restrict.
        Sign up now to experience the future of payments and offer your customers the best checkout experience.
      ul.list
        li Accept UPI payments for FREE
        li Personalised QR with your business logo
        li Generate unlimited QRs. No restrictions.
        li SMS and email alerts for all payments
      +button-anchor(text='Sign Up' class="signupCTA"
        data-lytics="click"
        data-lytics-category="Website - Rzp Bottom"
        data-lytics-action="Click - Signup"
         data-location="footer"
        arrow
        small) 

append scripts
  script(src='/qr-code.js')
  script(src='https://www.google.com/recaptcha/api.js' defer)

![Screenshot 2021-12-03 at 7 22 34 PM](https://user-images.githubusercontent.com/10862593/144613685-a17b8bed-2a34-495a-b7c8-65c13c859573.png)